modules/manage/pages/iceberg/iceberg-topics-databricks-unity.adoc

```diff
@@ -27,13 +27,9 @@ endif::[]
 * Workspace admin privileges to complete the steps to create a Unity Catalog storage credential and external location that connects your cluster's Tiered Storage bucket to Databricks.
 
 == Limitations
+ 
+The following data types are not currently supported for managed Iceberg tables:
```

Comment:
@nvartolomei this looks correct me to based on recent slack convos but can you take a quick 👀 , please?